themes/socialbase/templates/comment/comment--activity.html.twig

```diff
@@ -66,9 +66,8 @@
  * @ingroup themeable
  */
 #}
-{{ content.commented_entity }}
-{{ content.comment_link }}
-<div class="card__nested-section">
+
```

Comment:
@rollki Did we check if after removing those this kind of comments looks good in default activity streams in both themes?